.gitignore

```diff
@@ -6,6 +6,9 @@ bench_verify
 bench_schnorr_verify
 bench_recover
 bench_internal
+bench_generator
```

Comment:
since you're adding `bench_generator', you could also add the missing `bench_whitelist`